extras/po/fr.po

```diff
@@ -882,7 +882,7 @@ msgstr ""
 #: tomb:Open:mount_tomb:1830
 msgid "Last visit by ::1 user::(::2 tomb build::) from ::3 tty:: on ::4 host::"
 msgstr ""
-"Dernière visite par ::1 user::(::2 tomb build) depuis ::3 tty:: sur ::4 "
+"Dernière visite par ::1 user::(::2 tomb build::) depuis ::3 tty:: sur ::4 "
 "host::"
```

Comment:
I can't remember if it's POEditor or `msgmerge`, but one on them is splitting the lines every time I update translations, so it better stays like that.